src/openms/include/OpenMS/ANALYSIS/ID/SiriusAdapterAlgorithm.h

```diff
@@ -73,6 +73,14 @@ namespace OpenMS
 
       };
 
+      /**
+      @brief Comparison function using the extracted scan_index from the sirius workspace file path
+
```

Comment:
so `i` and `j` are scan_indices or file paths?
maybe make that more clear and use variable names which reflect that
`static bool extractAndCompareScanIndexLess(const String& filepath_1, const String& filepath_2);`

Reply:
Ok, thanks.

`i` and `j` are the scan indices, which are extracted from the file paths in the sirius workspace.

`ws_known/sirius_out/2026_20200930_173624_abinode00.informatik.uni-tuebing_161994086002579952-880005--9645--LysoPC1819Z` --> 9645

`ws_known/sirius_out/2026_20200930_175021_abinode00.informatik.uni-tuebing_5289776317112608729-1041006--11475--Pentacosano` --> 11475